### PR TITLE
fix(#165): nested query resolvers

### PR DESCRIPTION
**Pull request by yjkimjunior**
#165

### CODE REVIEW

**Comment by Tbaut** [comment]
This adds most of the changes that were merged recently, but what was the fix for nested queries?

**Comment by yjkimjunior** [comment]
https://github.com/paritytech/Nomidot/pull/168/files#diff-09b77f4cdfa212a035a21a8eef16bbb6

**Comment by yjkimjunior** [comment]
https://www.prisma.io/tutorials/a-guide-to-common-resolver-patterns-ct08/

**Comment by yjkimjunior** [comment]
i'm not done but that's the working fix for total issuances at least.

**Comment by yjkimjunior** [comment]
Screenshot 2020-02-13 at 14 15 30

**Comment by Tbaut** [comment]
ah yeah, understood, it'll be a little error prone to have to add that all! I've quite a lot of nested stuff.

**Comment by yjkimjunior** [comment]
yeah...sigh, it is what it is 🗻

File: back/server/prisma/datamodel.prisma [hunk 1]
```diff
@@ -65,6 +81,35 @@ type Stake {
...
+  memberCount: Int!
+  metaDescription: String!
+  method: String!
+  motionProposalArguments: [MotionProposalArgument]! @relation(name: "ArgumentForMotionProposal")
+  motionProposalHash: String!
+  motionProposalId: Int! @unique
+  preimage: Preimage @relation(name: "PreimageForMotion")
+  preimageHash: String
+  section: String!
+  status: [MotionStatus]! @relation(name: "MotionStatusFromMotion")
```
Comment (Tbaut) [inline back/server/prisma/datamodel.prisma]: Hmm this datamodel was actually changed in #167 
That's the "conflicts" I was talking about. In the end it's not a conflict, just something that needs to be synced.

**Comment by yjkimjunior** [comment]
no that's all you need to do. I would suggest the common troubleshooting steps though like clear db, redeploy model, re-get schema

**Comment by yjkimjunior** [comment]
https://recordit.co/KyX2HWiT7w/gif/

**Pull request merged**